cmd/kubeadm/.import-restrictions

```diff
@@ -0,0 +1,179 @@
+{
+	"Rules": [
+		{
+			"SelectorRegexp": "k8s[.]io/(api/|apimachinery/|apiextensions-apiserver/|apiserver/)",
+			"AllowedPrefixes": [
+				""
+			]
+		},
+		{
+			"SelectorRegexp": "k8s[.]io/client-go/",
```

Comment:
This syntax is most probably right but looks strange to me. Intuitively I would have expected something like `k8s\.io\/client-go\/.*`

Reply:
seems to work, followed the same patter as in other .import-restrictions. Also tested by hand using

- https://play.golang.org/p/3icjV53V8it
- https://regex-golang.appspot.com/assets/html/index.html